### PR TITLE
[WIP] Add encryption process

### PR DESCRIPTION
**Pull request by alexbagirov**
Refs #3

### CODE REVIEW

File: installer.py [hunk 1]
```diff
@@ -1,45 +1,62 @@
...
+        c = self.db.cursor()
+        
         c.execute('''CREATE TABLE users (name text PRIMARY KEY,
-                                         password text,
-
-                                         friends text ARRAY,
-                                         favorites text ARRAY,
-                                         blacklist text ARRAY,
-                                         dialogs text ARRAY)''')
+                                              password text,
```
Comment (illright) [inline installer.py]: What's up with indentation here?

Comment (alexbagirov) [reply installer.py]: Thanks!

It looks better now :+1:

**Review by illright — APPROVED** [review]

**Comment by alexbagirov** [comment]
What we need here is to write some tests :smile::
- [Flask testing](http://flask.pocoo.org/docs/0.11/testing/)
- [unittest](https://docs.python.org/3/library/unittest.html)

**Review by illright — APPROVED** [review]

**Review by illright — CHANGES_REQUESTED** [review]
Remove the `templates` folder

**Review by illright — APPROVED** [review]

**Pull request closed without merging**